tasks.gradle

```diff
@@ -25,6 +25,20 @@ task aggregateJavadoc(type: Javadoc) {
   source subprojects.javadoc.source
   destinationDir project.rootProject.file("site/docs/javadoc/${getJavadocVersion()}")
   classpath = project.rootProject.files(subprojects.javadoc.classpath)
+
+  final JAVADOC_FIX_SEARCH_STR = '\n\n' +
+          'getURLPrefix = function(ui) {\n' +
+          '    return \'\';\n' +
+          '};\n'
+
+  doLast {
+    // Fix bug with search
+    if (JavaVersion.current() >= JavaVersion.VERSION_11) {
```

Comment:
Maybe we should just require Javadoc to be built with Java 8 instead?

Reply:
The search is actually very nice addition, we can remove this fix and fix each time manually

Reply:
Oh, the search is only available with Java 11? Then that's a nice reason to keep it as you have it here.

Reply:
the search was added starting with Java 9, it became default in Java 10 (see [here](http://openjdk.java.net/jeps/224))